test/optimus/4kpbmc/ValidateOptimus.wdl

```diff
@@ -90,10 +96,12 @@ task ValidateBam {
 }
 
 task ValidateLoom {
-    File loom_file
-    String expected_loom_file_checksum
+    input {
+        File loom_file
```

Comment:
does this need to be optional?

Reply:
Yes well spotted, thank you. I really I forgot to make sure that the 4kpmbc test also runs correctly, will amend